sdcm/cluster.py

```diff
@@ -404,8 +404,12 @@ def is_client_encrypt(self):
 
     @property
     def cpu_cores(self):
-        result = self.remoter.run("nproc", ignore_status=True)
-        cores = result.stdout.lower()
+        try:
+            result = self.remoter.run("nproc", ignore_status=True)
+            cores = result.stdout.lower()
```

Comment:
Sorry for not pointing out it on first review.
Let's cover case when stdout is empty:

```
        try:
            result = self.remoter.run("nproc", ignore_status=True)
            return int(result.stdout)
        except Exception as details:
            self.log.error("Failed to get number of cores due to the: %s", details)
        return None
       
```

Reply:
done